vortex-array/src/stats/mod.rs

```diff
@@ -237,6 +244,14 @@ impl dyn Statistics + '_ {
             })
     }
 
+    pub fn get_as_bound<S, U>(&self) -> Option<S::Bound>
```

Comment:
I don't like that this takes a generic `S`, but other functions take a `Stat` enum? Can everything be done with generics? I guess not really...